Dockerfile

```diff
@@ -112,7 +112,11 @@ ARG odoo_version
 
 # Install Odoo requirements (use ADD for correct layer caching).
 # We use requirements from OCB for easier maintenance of older versions.
-ADD https://raw.githubusercontent.com/OCA/OCB/$odoo_version/requirements.txt /tmp/ocb-requirements.txt
+RUN set -eux; \
+    SHA=$(curl -s https://api.github.com/repos/OCA/OCB/branches/$odoo_version \
+        | grep '"sha"' | head -n 1 | cut -d '"' -f 4); \
+    curl -sSL "https://raw.githubusercontent.com/OCA/OCB/${SHA}/requirements.txt" \
+        -o /tmp/ocb-requirements.txt
```

Comment:
This needs to remain a ADD, otherwise the docker build will not detect a new version and use the cached layers.

Reply:
Ok, If I use the ADD on the branch is it ok for you?